### ISSUE TITLE
Drivetrain, Pneumatics, RobotContainer & Cameras - Week Zero

### ISSUE THREAD

**Issue by TylerSeiford** [FRCTeam2501/2020-Robot#13]
Overall:
1. Switch to tabs instead of spaces

Cameras:
1. The constants should be moved to their own sub namespace under CONSTANTS
2. We probably want the ability to have 2 cameras, one for looking forward and one for looking up at the goal.  With that will be a lot of work, so probably post week zero.

~~Climber:~~
~~1. The subsystem needs a second state machine to change which winch is being run.  This is a fairly large feature so I'll open up a separate issue for this.~~
~~2. You didn't destruct any objects from Climber, uh oh leaky...~~
~~3. ToggleWinchOn/Off/DownOn/DownOff need to be written.~~

Drivetrain:
1. AutoDrive can be removed.

Pneumatics:
1. TogglePneu/2 needs to be deleted.

Intake, Shooter and Hopper I'll review from Tony, Jacob, and Tony respectively (more issues on github yay).

RobotContainer:
1. Joysticks should be using ports.h not the dumb old example code
2. TogglePneu should be redone using the respective climber subsystem and an inlined instant command.
3. I think there are some buttons and features missing, I'll check this once the rest is done (namely the climber stuff).
4. The Periodic should be empty (that was my debugging).

**Comment by TylerSeiford** [comment]
I am going to move the Climber into it's own issue.

**Comment by TylerSeiford** [comment]
I fixed some of this in https://github.com/FRCTeam2501/2020-Robot/commit/06b0dfdc411cc091ceb88dcf81433cecd003b171, https://github.com/FRCTeam2501/2020-Robot/commit/ff157abb81aa9852e17400aa07acc9051428e265, https://github.com/FRCTeam2501/2020-Robot/commit/8b498e68a8a32c00adb6b8ffa26bb845fc3697bf, and https://github.com/FRCTeam2501/2020-Robot/commit/ebf8e4d9accfe5f53a0b8fea745048b341feaebd, but this still needs some more work.
